portfolio/src/main/webapp/style.css

```diff
@@ -386,6 +386,35 @@ p.account-username {
 }
 
 .comment-section-container {
+  height: 800px;
+}
+
+.form-container {
+  background-color: #fbfbfb;
+  height: 800px;
+  width: 50%;
+}
+
+.comment-feed-container {
   background-color: #ffc857;
+  display: flex;
+  flex-direction: column;
   height: 800px;
+  width: 50%;
+}
+
+p.user-comment {
+  background-color: #fbfbfb;
+  border: 5px solid #b8f2e6;
+  border-radius: 25px;
+  color: #FFA69E; 
+  font-family: 'Judson', serif;
+  font-size: 40px;
+  margin-bottom: 5px;
```

Comment:
You can use the shorthand to combine these margin rules

```css
/* margin: vertical  horizontal */
margin: 5px 10px;
```

https://developer.mozilla.org/en-US/docs/Web/CSS/margin

Reply:
adjusted